### PR TITLE
Drop global HOME environment variable

### PR DESCRIPTION
**Pull request by TobiX**
HOME is normally controlled by the "system"[1] and should match what the user's home directory in `/etc/passwd` is. Overwriting this value with `ENV` in this docker image causes some surprises when the user is changes with `USER` in a docker image derived `FROM` this image. Unfortunatly, docker still doesn't allow to "reset" environment variables (https://github.com/moby/moby/issues/3465), so if I want to change the user, I need to remember to also update `HOME`.

Long story short: This removes the global preset for `HOME`, so `HOME` does function as expected.

[1] I think in normal UNIX sessions, login is responsible for that, but in the case of docker, docker itself takes care of that..

### CODE REVIEW

**Comment by slide** [comment]
Close and reopen for build

**Comment by oleg-nenashev** [comment]
i will do a new release once we have a new Remoting version. CC @jeffret-b

**Comment by jeffret-b** [comment]
I'm hoping to get a new Remoting version out the first half of next week.
